frontend/src/components/AutoJoinEmailsInput/index.tsx

```diff
@@ -0,0 +1,98 @@
+import { useAuthContext } from '@authentication/AuthContext'
```

Comment:
This was a copy/paste mostly from the original component. Designs are just Julianified.